angular/playground/projects/ng-forms-manager/src/lib/forms-manager.ts

```diff
@@ -156,20 +172,30 @@ export class AkitaNgFormsManager<FormsState = any> {
     this.unsubscribe(formName);
   }
 
-  unsubscribe(formName?: keyof FormsState, config: { removeNgForm?: boolean } = {}) {
-    const _config = { removeNgForm: true, ...config };
+  unsubscribe(formName?: keyof FormsState, config: { removeNgForm?: boolean; updateStore?: boolean } = {}) {
+    const _config = {
+      removeNgForm: true,
+      ...{ updateStore: this.options.updateStoreOnUnsubscribe },
+      ...config
+    };
     const _formName = formName as any;
     const removeInstance = (name: any) => (_config.removeNgForm ? this.removeFormInstance(name) : null);
 
     if (_formName) {
       if (this.valueChanges[_formName]) {
         this.valueChanges[_formName].unsubscribe();
         delete this.valueChanges[_formName];
+        if (config.updateStore) {
```

Comment:
Note that this will work only if you set `persistForm` to `true`.

Reply:
Added a check to see if the form manager has this form before updating.